src/doc/sphinx_cli_extractor.py

```diff
@@ -402,7 +410,23 @@ def __str__(self):
         lines = ""
         for i in range(len(self.names)):
             name    = "%s" % (self.names[i].strip())
-            lines += "\n%s : type\n    %s\n"%(name, self.descriptions[i])
+            if name == 'none':
+                continue
+                
+            #
+            # Attempt to determine the type of argument from the argument's
+            # description.
+            #
+            type_name = "STARTING_VALUE"
```

Comment:
Ok, now I understand why I see some instances of this in the `.rst` file. Didn't seem like it happened a lot sot patching up manually after the switchover seems fine to me.